index.html

```diff
@@ -1,12 +1,250 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Esport Events</title>
+    <!-- <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/tooltipster/2.3.0/css/reset.min.css"> -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+    <link rel="stylesheet" href="assets/css/index.css">
+    
+    
 </head>
+
 <body>
-    
+    <nav id="navbar-desk" class="nav-desk isDesktop">
+        <div class="dark-bar isDesktop">
+            <div class="bar-content">
+                <a href="#"><i class="fa-brands fa-facebook-f"></i></a>
+                    <a href="#"><i class="fa-brands fa-twitter"></i></a>
+            <a href="#">English</a>
+            <a href="#">My Page</a>
+            <a href="#">Login</a>
+        </div>
+        </div>
+        <div class="desktop-nav isDesktop" >
+           <a href="/index.html" class="head-logo-link"> <div class="head-logo">
+                <i class="fa-solid fa-gamepad fa-2xl logo-icon"></i> <h5 class="title">E-Sport <br> Events</h5>
+            </div></a>
+            <ul class="nav-links-desk">
+                <li><a class="orangecolor" href="/pages/about.html">About</a></li>
+                <li><a href="/index.html#main-program">Program</a></li>
+                <li><a href="/index.html#FeaturedSpeakers">Speakers</a></li>
+                <li><a href="/index.html#sponsors">Partners</a></li>
+            </ul>
+            <a class="main-btn" href="#">EE Compaign</a>
+        </div>
+        </nav>
+    <header>
+        <div class="mobile-menu-container hide isMobile">
+           <div class="close-icon" id="close-mobile-menu">
+            <i class="fa-solid fa-xmark fa-xl"></i>
+           </div>
+            <div class="mobile-menu">
```

Comment:
- [ ] **Required:** Kindly replace most of your `div`s with more semantic tags. This helps SEO listings.

![image](https://user-images.githubusercontent.com/27709832/163198753-efa63fab-b38b-4971-9bfb-7fabe7a26d46.png)